sdk/formrecognizer/azure-ai-formrecognizer/samples/async_samples/sample_recognize_receipts_async.py

```diff
@@ -0,0 +1,72 @@
+# coding: utf-8
```

Comment:
It would be great if this could come first in the directory listing of samples, as it should work out of the box.

Reply:
Ok, I had moved it lower than recognize_custom_forms since we agreed that the ordering of FRClient's methods was 
1. custom forms
2. content
3. recognize receipts.

I'll switch the ordering in the samples to have 
1. recognize content (since this should also work out the gate)
2. recognize receipts
3. recognize custom forms

Reply:
Yes, sorry for being contradictory here.  I think for README concepts, custom forms should come first as it's the service's primary use case, but for samples, things that work out of the box should come first.